lib/generator.js

```diff
@@ -17,13 +17,21 @@ TemplateGenerator.prototype.render = function render(mail, cb) {
     if (mail.data.html) return cb();
 
     var templatePath = path.join(this.viewPath, mail.data.template + this.extName);
-
+    var text_templatePath='';
+    if (mail.data.text_template!==undefined)text_templatePath=path.join(this.viewPath, mail.data.text_template + this.extName);
```

Comment:
This should probably also not trigger if the option is set to null. `if (mail.data.textTemplate != null)` should handle both undefined and null.

Reply:
Additionally please separate this onto multiple lines and using spacing to make this more readable.